docs/configuration/appointment.rst

```diff
@@ -0,0 +1,51 @@
+.. _configuration_appointment:
+
+=========================
+Appointment configuration
+=========================
+
+Appointments can be made using two separate services.  JCC and Qmatic.
+Open Forms can be configured to make use of one of these services.
+
+JCC
+---
+
+1. In Open Forms navigate to: **Configuration** > **JCC configuration**
+2. Click the **Green Plus Button** and fill in the following details:
+
+   * **Label**: JCC
+   * **Ontvangende organisatie**: Open Forms
+   * **Ontvangende applicatie**: Open Forms
+   * **Versturende organisatie**: Horstaandemaas
+   * **Versturende applicatie**: Afspraak Acceptatie
+   * **URL**: https://afspraakacceptatie.horstaandemaas.nl/JCC/Horst%20aan%20de%20Maas/WARP/GGS2/GenericGuidanceSystem2.asmx?wsdl
+
+3. Click **Save**
+
+Qmatic
+------
+
+1. Obtain a `Qmatic API key`_ from Qmatic.
+2. In Open Forms navigate to: **Configuration** > **Qmatic configuration**
+3. Click the **Green Plus Button** and fill in the following details:
+
+   * **Label**: Qmatic
+   * **Type**: ORC (Overige)
+   * **API root URL**: ``TODO: FILL ME IN``
+   * **Authorization type**: API key
+   * **Header key**: X-Api-Key
+   * **Header value**: *The Qmatic API key obtained in step 1*
+   * **OAS**: ``TODO: FILL ME IN``
```

Comment:
Can you verify this is correct and add the correct urls for the `API root URL` and `OAS`.

Reply:
No clue, and the they differ I think per organisation.